### PR TITLE
Support MobileOne on ONNXRuntime and TensorRT

### PR DESCRIPTION
**Pull request by SsTtOoNnEe**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Based on branch `dev-1.x`, `MobileOne` (in MMClassification) has been supported on ONNXRuntime and TensorRT backend.

## Modification

Updated document and test ymal file.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by lvhan028** [comment]
Please don't check in converted models

File: modules_statistic_results.json [hunk 1]
```diff
@@ -0,0 +1,578 @@
+{
+  "scan_date": "2022-10-28 13:33:21",
```
Comment (RunningLeon) [inline modules_statistic_results.json:2]: pls remove this file.

File: tests/regression/mmcls.yml [hunk 2]
```diff
@@ -211,3 +211,15 @@ models:
...
       - *pipeline_ort_dynamic_fp32
       - *pipeline_trt_dynamic_fp32
+
+  - name: MobileOne
+    metafile: configs/mobileone/metafile.yml
+    model_configs:
+      - configs/mobileone/mobileone-s0_8xb128_in1k.py
+    pipelines:
+      - *pipeline_ort_dynamic_fp32
+      - *pipeline_ort_static_fp32
```
Comment (RunningLeon) [inline tests/regression/mmcls.yml:221]: only keep static with fp16

File: tests/regression/mmcls.yml [hunk 3]
```diff
@@ -211,3 +211,10 @@ models:
       - *pipeline_ts_fp32
       - *pipeline_ort_dynamic_fp32
       - *pipeline_trt_dynamic_fp32
+
+  - name: MobileOne
+    metafile: configs/mobileone/metafile.yml
+    model_configs:
+      - configs/mobileone/mobileone-s0_8xb128_in1k.py
+    pipelines:
+      - *pipeline_trt_static_fp16
```
Comment (RunningLeon) [inline tests/regression/mmcls.yml:220]: could add pipeline_ort_dynamic_fp32 since it supported onnxruntime

Comment (RunningLeon) [reply tests/regression/mmcls.yml:220]: @SsTtOoNnEe

**Comment by RunningLeon** [comment]
@SsTtOoNnEe could fix conflicts

File: tests/regression/mmcls.yml [hunk 4]
```diff
@@ -210,6 +210,15 @@ models:
...
+      - *pipeline_trt_dynamic_fp32
+
+  - name: MobileOne
+    metafile: configs/mobileone/metafile.yml
+    model_configs:
+      - configs/mobileone/mobileone-s0_8xb128_in1k.py
+    pipelines:
+      - *pipeline_trt_static_fp16
+      - *pipeline_ort_dynamic_fp32
       - *pipeline_trt_static_fp16
```
Comment (RunningLeon) [inline tests/regression/mmcls.yml:222]: @SsTtOoNnEe  this pipeline is of `SwinTransformer`, pls. added it back

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by hanrui1sensetime — APPROVED** [review]
Fix conflict, then LGTM.

**Comment by RunningLeon** [comment]
@SsTtOoNnEe Pls. fix lint error

**Pull request merged**